### PR TITLE
Adds a post that shows how to optimize Stable Diffusion for Intel CPUs even further

### PR DESCRIPTION
**Pull request by sayakpaul**
Internal draft: https://docs.google.com/document/d/1yloJd6IH6Pvj_NNbjcdKjzdYvXdGlywqCEsR0YpBQBk/edit

Doesn't need further reviews. 

Cc: @mfuntowicz @jeffboudier @AlexKoff88 @osanseviero 

## TODOs:

- [x] Amend the current draft as per markdown formatting
- [x] Thumbnail
- [x] Make the necessary changes in the files in `blog`
- [x] Host the images under https://huggingface.co/datasets/huggingface/documentation-images/ and replace the image URLs

### CODE REVIEW

**Comment by sayakpaul** [comment]
@osanseviero I preview the draft using Codespaces. 

I am not sure why @ @AlexKoff88 is not showing up here:

Screenshot 2023-05-24 at 10 02 13 AM

Is there anything I am missing out on?

File: train-optimize-sd-intel.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+ 
+We modified the Token Merging method to be compliant with OpenVINO and stacked it with 8-bit quantization when applied to the Attention UNet model. This also includes all the mentioned techniques including Knowledge Distillation, etc. As for quantization, it requires fine-tuning to be applied to restore the accuracy. We also start optimization and fine-tuning from the [model](https://huggingface.co/svjack/Stable-Diffusion-Pokemon-en) trained on the [Pokemons dataset](https://huggingface.co/datasets/lambdalabs/pokemon-blip-captions). The figure below shows an overall optimization workflow.
```
Comment (AlexKoff88) [inline train-optimize-sd-intel.md:50]: "...includes...including" sounds not good. Maybe rephrase is as "...involves...including"

File: train-optimize-sd-intel.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+pipe.reshape(batch_size=1, height=512, width=512, num_images_per_prompt=1)
+pipe.compile()
+
+# Generate an image.
+prompt = "a drawing of a green pokemon with red eyes"
+output = pipe(prompt, num_inference_steps=50, output_type="pil").images[0]
+output.save("image.png") 
+```
+
+You can find the training and quantization [code](https://github.com/huggingface/optimum-intel/tree/main/examples/openvino/stable-diffusion) in the Hugging Face Optimum-Intel library. The notebook that demonstrates the difference between optimized and original models is available [here](https://github.com/huggingface/optimum-intel/blob/main/notebooks/openvino/stable_diffusion_optimization.ipynb). You can also find a bunch of models in our [group](https://huggingface.co/OpenVINO) on the Hugging Face Hub. We have also created a [demo](https://huggingface.co/spaces/AlexKoff88/stable_diffusion) on Hugging Face Spaces that is being run on r6id.2xlarge instance with 3rd Generation Intel Xeon Scalable processor.
```
Comment (AlexKoff88) [inline train-optimize-sd-intel.md:89]: "We have also created a [demo]...". "also" is used in the previous sentence. I would rephrase it to "In addition, we have created..." for better reading.

File: train-optimize-sd-intel.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+| ![](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/train-optimize-sd-intel/image_fp32.png) | OpenVINO FP32 | 120 seconds <br>**(1.9x)** | 3.44 GB |
+| ![](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/train-optimize-sd-intel/image_quantized.png) | OpenVINO 8-bit | 59 seconds<br>**(3.9x)** | 0.86 GB<br>**(0.25x)** |
+| ![](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/train-optimize-sd-intel/image_tome_quantized.png) | OpenVINO ToMe + 8-bit | 44.6 seconds<br>**(5.1x)** | 0.86 GB<br>**(0.25x)** |
+
+Results of image generation [demo](https://huggingface.co/spaces/AlexKoff88/stable_diffusion) using different optimized models. Input prompt  is “cartoon bird”, seed is 42. The models are with OpenVINO 2022.3 in [Hugging Face Spaces](https://huggingface.co/spaces/AlexKoff88/stable_diffusion) using a “CPU upgrade” instance which utilizes 3rd Generation Intel® Xeon® Scalable Processors with Intel® Deep Learning Boost technology.
```
Comment (helena-intel) [inline train-optimize-sd-intel.md:63]: Do Hugging Face Spaces now use Ice Lake Xeon?

Comment (sayakpaul) [reply train-optimize-sd-intel.md:63]: Cc: @julien-c maybe.

**Comment by osanseviero** [comment]
@sayakpaul regarding

> @osanseviero I preview the draft using Codespaces.
> I am not sure why @ @AlexKoff88 is not showing up here:

In your codespace you might have a database copy without the other users. It should be ok once deployed.
